cmd/veneur-emit/main_test.go

```diff
@@ -504,8 +506,95 @@ func TestBuilldSCPacket(t *testing.T) {
 	})
 }
 
-func resetMap(m map[string]bool) {
-	for key := range m {
-		m[key] = false
-	}
+func TestInvocations(t *testing.T) {
```

Comment:
It's too bad we can't use write this as a testable example. Could we at least add a regular example file while we're at it? (We can reuse these exact invocations as demonstrations).

Reply:
you mean https://golang.org/pkg/testing/#hdr-Examples? yeah i can add some. i don't think we can meaningfully test the output though

Reply:
Yeah, we definitely can't test the output (which is a shame, but unavoidable). 


I also just realized that the `_example` file isn't going to be particularly helpful anyway, since this is intended to be used as a CLI utility and not a Go library. So what we have in the README is probably sufficient.